### PR TITLE
Add steps to enable Pagespeed

### PR DESCRIPTION
**Pull request by chipironcin**

### CODE REVIEW

**Comment by sameersbn** [comment]
test this please

File: README.md [hunk 1]
```diff
@@ -164,6 +164,39 @@ You can also reload the nginx configuration by sending the `HUP` signal to the c
 docker kill -s HUP nginx
 ```
 
+## Enabling Pagespeed module
+
+This image include Pagespeed module for Nginx.
+
+In order to activate it, mount the configuration volume following the steps in "Full Configuration" section above and edit the file located at `/path/to/nginx/conf/bitnami/bitnami.conf` adding the following snippet inside the `server` directive:
```
Comment (prydonius) [inline README.md]: We should be able to link to the full configuration section with [Full Configuration](#full-configuration)

File: README.md [hunk 2]
```diff
@@ -164,6 +164,39 @@ You can also reload the nginx configuration by sending the `HUP` signal to the c
 docker kill -s HUP nginx
 ```
 
+## Enabling Pagespeed module
+
+This image include Pagespeed module for Nginx.
```
Comment (prydonius) [inline README.md]: includes the

File: README.md [hunk 3]
```diff
@@ -164,6 +164,39 @@ You can also reload the nginx configuration by sending the `HUP` signal to the c
...
+    # needs to exist and be writable by nginx
+    pagespeed FileCachePath /installdir/nginx/var/ngx_pagespeed_cache;
+    location ~ "\.pagespeed\.([a-z]\.)?[a-z]{2}\.[^.]{10}\.[^.]+" { add_header "" ""; }
+    location ~ "^/ngx_pagespeed_static/" { }
+    location ~ "^/ngx_pagespeed_beacon$" { }
+    location /ngx_pagespeed_statistics { allow 127.0.0.1; deny all; }
+    location /ngx_pagespeed_message { allow 127.0.0.1; deny all; }
+```
+
+Then reload nginx configuration executing the following command
```
Comment (prydonius) [inline README.md]: Can we add the reloading command to the "Restart nginx" step above and link to that instead of repeating it?

**Pull request merged**